conn.go

```diff
@@ -25,25 +24,25 @@ func beginHandlerOpen(ch chan error, removeHandlers func()) func(ev *js.Object)
 
 // closeError allows a CloseEvent to be used as an error.
 type closeError struct {
-	*dom.CloseEvent
+	*js.Object
 }
 
 func (e *closeError) Error() string {
 	var cleanStmt string
-	if e.WasClean {
+	if e.Get("wasClean").Bool() {
 		cleanStmt = "clean"
 	} else {
 		cleanStmt = "unclean"
 	}
-	return fmt.Sprintf("CloseEvent: (%s) (%d) %s", cleanStmt, e.Code, e.Reason)
+	return fmt.Sprintf("CloseEvent: (%s) (%d) %s", cleanStmt, e.Get("code"), e.Get("reason"))
```

Comment:
`e.Get("code")` should be `e.Get("code").Int()` and `e.Get("reason")` should be `e.Get("reason").String()`.

That way, you get correct error code printed:

![image](https://cloud.githubusercontent.com/assets/1924134/14073541/c6965688-f47e-11e5-89fd-c9a73bfdfa8f.png)

Before this change, it was not printing the right thing:

![image](https://cloud.githubusercontent.com/assets/1924134/14073562/f070d820-f47e-11e5-903e-4c8f6fa1c72e.png)

(The `Get("reason").String()` part may not strictly need `.String()`, but better to do it for consistency.)